CHANGELOG.md

```diff
@@ -213,27 +256,37 @@ Existing identities are not affected by this change.
 
 ### fix: Certification for aliasing updates on asset deletion
 
-Best explained by an example: Two assets exist with aliasing enabled: `/content` and `/content.html`. Usually, when requesting `/content`, `/content.html` is served because it has aliasing enabled.
-But in this scenario, because `/content` exists, it overwrites the alias and `/content` is served when requesting the path `/content`.
+Best explained by an example: Two assets exist with aliasing enabled: `/content` and `/content.html`. Usually, when
```

Comment:
Is all this reformatting of the changelog intentional?

Reply:
was not. my ide does this automatically when I commit. I'll revert